challenge_scoring/utils/attributes.py

```diff
@@ -51,7 +51,6 @@ def compute_attrib_files(root_dir, attrib_func, attrib_name):
     fnames = glob.glob(os.path.join(root_dir, '*'))
```

Comment:
Methods get_attribs_for_file, save_attribs , merge_attribs, compute_attrib_files do not seem to be used anymore.